packages/next/cli/next-build.ts

```diff
@@ -8,18 +8,23 @@ import build from '../build'
 import { printAndExit } from '../server/lib/utils'
 
 const nextBuild: cliCommand = (argv) => {
-  const args = arg(
-    {
-      // Types
-      '--help': Boolean,
-      '--profile': Boolean,
-      // Aliases
-      '-h': '--help',
-    },
-    { argv }
-  )
-
-  if (args['--help']) {
+  const validArgs: arg.Spec = {
+    // Types
+    '--help': Boolean,
+    '--profile': Boolean,
+    // Aliases
+    '-h': '--help',
+  }
+
+  let args: arg.Result<arg.Spec>
+  try {
+    args = arg(validArgs, { argv })
+  } catch (error) {
```

Comment:
I think if you write this as
```js
  } catch (error) {
    if (error.code === 'ARG_UNKNOWN_OPTION') {
      return printAndExit(error.message, 1)
    }
    throw error
  }
```
typescript will be able to correctly infer the `args` type later on, so you won't have to use `args!`

Reply:
Oh! Thanks :D